pkgs/by-name/li/libitl/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libitl";
+  version = "0.1.8";
```

Comment:
Can you clarify how this version number was selected? It doesn't seem to match any upstream tags or information.

If upstream does not have a reliable, maintained versioning scheme, and you are simply following the master branch (Which appears to be the case), then you should consider using the `unstable-<date>` format instead.

Reply:
The `CMakeLists.txt` has a version number, which is where I got that from.

Reply:
(And note, this also thus results in the main shared library output being `libitl.so.0.1.8`)

Reply:
Version should point to an upstream Git tag, if it's not present or if it's way too old, you can pin src to a specific commit, but then you'll have to set version as `VERSION-unstable-YYYY-MM-DD`, more on that [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming).